amaranth_boards/gatemate_a1_evb.py

```diff
@@ -0,0 +1,305 @@
+            Subsignal("fwd", Pins("IO_WA_B5", dir="o")),
+        ),
+
+        # TODO: Check how to declare LVDS pins
+        # TODO: Check direction of serdes pins
+        Resource("serdes", 0,
+            Subsignal("ser_clk_n", Pins("IO_SER_CLK_N", dir="io")),
+            Subsignal("ser_clk_p", Pins("IO_SER_CLK_P", dir="io")),
+            Subsignal("ser_tx_p",  Pins("IO_SER_TX_P",  dir="io")),
+            Subsignal("ser_tx_n",  Pins("IO_SER_TX_N",  dir="io")),
+            Subsignal("ser_rx_n",  Pins("IO_SER_RX_N",  dir="io")),
+            Subsignal("ser_rx_p",  Pins("IO_SER_RX_P",  dir="io"))
+        ),
+
+        Resource("uext", 0,
+            Subsignal("txd",  Pins("IO_EA_A0", dir ="o" )),
+            Subsignal("rxd",  Pins("IO_EA_B0", dir ="i" )),
+            Subsignal("scl",  Pins("IO_EA_A1", dir ="io")),
+            Subsignal("sda",  Pins("IO_EA_B1", dir ="io")),
+            Subsignal("miso", Pins("IO_EA_A2", dir ="i" )),
+            Subsignal("mosi", Pins("IO_EA_B2", dir ="o" )),
+            Subsignal("sck",  Pins("IO_EA_A3", dir ="o" )),    
+            Subsignal("cs",   Pins("IO_EA_B3", dir ="o" )),
+        ),
+
+        Resource("pmod", 0,
+            Subsignal("1",  Pins("IO_EA_A4", dir ="io")),
+            Subsignal("7",  Pins("IO_EA_B4", dir ="io")),
+            Subsignal("2",  Pins("IO_EA_A5", dir ="io")),
+            Subsignal("8",  Pins("IO_EA_B5", dir ="io")),
+            Subsignal("3",  Pins("IO_EA_A6", dir ="io")),
+            Subsignal("9",  Pins("IO_EA_B6", dir ="io")),
+            Subsignal("4",  Pins("IO_EA_A7", dir ="io")),    
+            Subsignal("10", Pins("IO_EA_B7", dir ="io")),
+        ),
+        
+
+        # Mostly used for RP2040
+        Resource("gpio", 0,
+            Subsignal("0",  Pins("IO_SA_A0", dir ="io")),
+            Subsignal("1",  Pins("IO_SA_B0", dir ="io")),
+            Subsignal("2",  Pins("IO_SA_A1", dir ="io")),
+            Subsignal("3",  Pins("IO_SA_B1", dir ="io")),
+            Subsignal("4",  Pins("IO_SA_A2", dir ="io")),
+            Subsignal("5",  Pins("IO_SA_B2", dir ="io")),
+            Subsignal("6",  Pins("IO_SA_A3", dir ="io")),
+            Subsignal("7",  Pins("IO_SA_B3", dir ="io")),
+            Subsignal("8",  Pins("IO_SA_A4", dir ="io")),
+            Subsignal("9",  Pins("IO_SA_B4", dir ="io")),
+            Subsignal("10", Pins("IO_SA_A5", dir ="io")),
+            Subsignal("11", Pins("IO_SA_B5", dir ="io")),
+            Subsignal("14", Pins("IO_SA_A7", dir ="io")),
+            Subsignal("15", Pins("IO_SA_B7", dir ="io")),
+            Subsignal("21", Pins("IO_SB_B8", dir ="io")), # GPIN1
+            Subsignal("26", Pins("IO_SB_B4", dir ="io")),
+            Subsignal("27", Pins("IO_SB_A4", dir ="io")),
+            Subsignal("28", Pins("IO_SB_A8", dir ="io")),
+            Subsignal("29", Pins("IO_SB_B8", dir ="io")),
+        ),
+    ]
+
+
+    connectors = [
+        Connector("bank_na1", 0, {        
```

Comment:
Isn't this just an enumeration of every pin the FPGA has?

Reply:
![PCB 1 top](https://github.com/user-attachments/assets/95164c37-02ba-4da6-9d8e-bfa744623fc0)
These are the free banks that have connectors: `Bank_NA1`, `Bank_NB1`, `Bank_EB1`, and `MISC1`.
I guessed that they should be in the connectors, here they are in the schematic.
![image](https://github.com/user-attachments/assets/dfb54757-8cd3-4b65-92b2-25a812353f80)

Reply:
OK, these should indeed be connectors.